ui/vite.config.ts

```diff
@@ -0,0 +1,139 @@
+import {defineConfig, normalizePath} from 'vite';
+import {dirname, resolve} from 'path';
+import {fileURLToPath} from 'url';
+import {viteStaticCopy} from 'vite-plugin-static-copy';
+import babel from '@rollup/plugin-babel';
+import inlinesvg from 'postcss-inline-svg';
+import cssimport from 'postcss-import';
+import postcssurl from 'postcss-url';
+
+
+// @ts-ignore
+const __dirname = dirname(fileURLToPath(import.meta.url));
+const cesiumBuild = resolve(__dirname, './node_modules/cesium/Build/Cesium');
+const extensions = ['.ts', '.js'];
+
+export default defineConfig({
+  resolve: {
+    alias: {
+      cesium: normalizePath(resolve(__dirname, 'node_modules/cesium')),
+      './cesium/Build': normalizePath(resolve(__dirname, 'node_modules/cesium/Build')),
+      './cesium': normalizePath(resolve(__dirname, 'node_modules/cesium/Source')),
+      './fomantic-ui-css': normalizePath(resolve(__dirname, 'node_modules/fomantic-ui-css')),
+      './@fontsource/inter': normalizePath(resolve(__dirname, 'node_modules/@fontsource/inter')),
+      'src': normalizePath(resolve(__dirname, 'src')),
+    },
+    extensions,
+  },
+  build: {
+    outDir: 'dist',
+    emptyOutDir: false,
+    minify: 'terser',
+    sourcemap: true,
+    cssCodeSplit: true,
+    rollupOptions: {
+      input: 'index.html',
+      output: {
+        entryFileNames: 'assets/[name]-[hash].js', // Ensure JavaScript output is configured

```

Comment:
This comment confuses me, if anything.
```suggestion
        entryFileNames: 'assets/[name]-[hash].js',configured
```